redcalendar/events/models.py

```diff
@@ -6,13 +6,11 @@
 
 # Create your models here.
 
-poster_storage = FileSystemStorage(location='./events/posters')
-
 class Event(models.Model):
     event_name = models.CharField(max_length=200)
     event_date = models.DateTimeField('date')
     event_place = models.CharField(max_length=100)
-    event_poster = models.ImageField(storage=poster_storage)
+    event_poster = models.ImageField(upload_to='posters')
```

Comment:
nothing wrong with `storage` what you used. But `upload_to` is easier and defaults to `I'll upload to MEDIA_ROOT + upload_to` in the filesystem More https://docs.djangoproject.com/en/1.7/ref/models/fields/#django.db.models.FileField.upload_to